### ISSUE TITLE
NTR: yokan and friends

### ISSUE THREAD

**Issue by pbuttigieg** [FoodOntology/foodon#30]
Greetings @Public-Health-Bioinformatics ! 
We  (@TaskeHAMANO and I) are aligning [MEO](https://bioportal.bioontology.org/ontologies/MEO) to ENVO and are reimporting many food terms into ENVO to support annotators asserting food environments for their metagenomics projects. 

We're going to have a number of requests coming up in the next few days. These will use East Asian characters in links, which may require some encoding updates.

The first set (which I'm not sure where to place), includes:

```
label: yokan
definition: A  jellied dessert made of red bean paste, agar, water, and sugar.
  dbxref for defintion: https://en.wikipedia.org/wiki/Y%C5%8Dkan
comment: The agar is usually in the form of kantens, agar sticks which require hydration.

label: neri yōkan
SubclassOf: yokan
definition: A yokan which has less water added to it during preparation, relative to standard yokan.
  dbxref for def: https://en.wikipedia.org/wiki/Y%C5%8Dkan

label: mizu yōkan
SubclassOf: yokan
definition: A yokan which has more water added to it during preparation, relative to standard yokan.
  dbxref for def: https://en.wikipedia.org/wiki/Y%C5%8Dkan

label: an
definition: A paste which is made from beans, grains, or other vegetables which are boiled, mashed, and diluted into a slurry which strained and then filtered, squeezed dry, and then usually sweetened. 
  dbxref for def: https://en.wikipedia.org/wiki/Sweet_bean_paste
  dbxref for def: https://ja.wikipedia.org/wiki/餡
narrow synonym: sweet bean paste
narrow synomyn:

label:sweet bean paste
SubClassOf: an
definition: An an which is made from beans.
  dbxref for def: https://en.wikipedia.org/wiki/Sweet_bean_paste
comment: There should be more sublcasses of this term created when needed, as many beans may be used.

label: azuki bean paste
SubClassOf: sweet bean paste
definition: A sweet bean paste which is made from Vigna angularis beans.
exact synonym: adzuki bean paste
exact synonym: red mung bean paste
exact synonym: aduki bean paste


label: shiro an
SubclassOf: sweet bean paste
definition:  A paste made of white kidney beans (from plants of the Phaseolus vulgaris species) and which has a milky and translucent appearance.
  dbxref for def: https://en.wikipedia.org/wiki/Y%C5%8Dkan
comment: Shiro an is sometimes used to make yokan, especially mizu yokan, however it is not, strictly speaking, yokan. Sugar is often added when this is used as a dessert ingredient or a dessert.
exact synonym: aduki bean paste
```

I think you can axiomatise the less and more water using "has [increased/decreased] levels of" and some wetness quality from PATO.

**Comment by pbuttigieg** [comment]
PS: For microcrediting:
My ORCID : http://orcid.org/0000-0002-4366-3088
@TaskeHAMANO 's ORCID :  http://orcid.org/0000-0002-1816-4260

**Comment by suzuki-shm** [comment]
@pbuttigieg 

I made a review and found one correction.
```
label: shiro an
...
exact synonym: aduki bean paste
```
shiro an is one of the bean paste, but aduki(adzuki) bean paste is used to express red bean paste. Thus exact synonym of shiro an would be "white bean paste".
Also, dbxref of azuki bean paste would be  https://en.wikipedia.org/wiki/Red_bean_paste . A short explanation is also written about white bean paste.

**Comment by Public-Health-Bioinformatics** [comment]
One thing I should note is that the experimental SIREN database of food products, which is waiting in the wings pending review and placement of items, is the import/subset_siren_import.owl file which is NOT included in the foodon.owl and foodon_core.owl files (which are currently the same).  It is in the foodon_siren.owl file.  Its history is that it is from LanguaL's SIREN database of indexed food products.  One issue with it is that none of the products have definitions.  But it does have products like red wine, white wine, coconut (food product) and over 9,000 others. I have set aside FoodOn ontology ids for this range of terms as indicated in the subset_siren_import.owl.  So in short, I'll be reusing the IDs already assigned for some of the above items in subset_siren_import.owl . 

But I REALLY appreciate having definitions, synonyms etc contributed as above for all the products!  I will implement them asap.

**Comment by Public-Health-Bioinformatics** [comment]
The above terms are now on github repo.  Haven't added the red bean paste dbxref but that will come soon. I assumed both ORCIDS would appear as definition editor on above items.

**Comment by Public-Health-Bioinformatics** [comment]
IDS:
FOODON_03309861
label: coconut product

FOODON_00002238
label: cocos fruit product

FOODON_00002243
label: nata de coco

FOODON_00002285
label: yokan

FOODON_00002382
label: neri yōkan

FOODON_00002383
label: mizu yōkan

FOODON_00002387
label: an


FOODON_00002388
label:sweet bean paste

FOODON_00002389
label: azuki bean paste

FOODON_00002390
label: shiro an

**Comment by pbuttigieg** [comment]
Thanks!

On 2 Oct 2017 20:45, "Damion Dooley" <notifications@github.com> wrote:

Closed #30 <https://github.com/FoodOntology/foodon/issues/30>.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/FoodOntology/foodon/issues/30#event-1275112375>, or mute
the thread
